recipe/meta.yaml

```diff
@@ -88,6 +90,8 @@ requirements:
     - mpc 1.1.*
     - mpfi 1.5.*
     - mpfr
+    # Since we apply 26326, we can only work with NetworkX 2.2
+    - networkx 2.2
```

Comment:
This is needed in run right?

Reply:
Oops, you're right.

Reply:
Instead of run, add it to `run_constrained`. Let me find an example for you